code/modules/mob/living/blood.dm

```diff
@@ -58,6 +58,10 @@
 			if(SPT_PROB(2.5, seconds_per_tick))
 				to_chat(src, span_warning("You feel [word]."))
 			adjustOxyLoss(round(0.005 * (BLOOD_VOLUME_NORMAL - blood_volume) * seconds_per_tick, 1))
+		if(BLOOD_VOLUME_SAFE to BLOOD_VOLUME_NORMAL)
+			if(SPT_PROB(2.1, seconds_per_tick))
```

Comment:
I don't like passive healing effects like this at all. If you get injured, you should go to medical. This is a free heal every 47 or so seconds and affects more than just being in crit.

I think you should just make it so that if you're in softcrit, you slowly heal towards 1 HP unless you have some wound. It'd work similar to epinephrine but it does not work if you have a wound or if you are past the softcrit threshold. Also make it significantly slowly that epinephrine so that the chem doesn't become useless.